build.gradle

```diff
@@ -1,19 +1,24 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
 
-  ext.KOTLIN_VERSION = "1.2.50"
+  ext.KOTLIN_VERSION = "1.2.51"
   ext.ANDROID_PLUGIN_VERSION = "3.2.0-beta02"
 
   repositories {
     google()
     jcenter()
   }
   dependencies {
-    classpath "com.android.tools.build:gradle:3.2.0-beta02"
+    classpath "com.android.tools.build:gradle:3.3.0-alpha04"
     classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$KOTLIN_VERSION"
     classpath 'com.bmuschko:gradle-nexus-plugin:2.3.1'
   }
 }
+
+plugins {
+  id 'com.github.ben-manes.versions' version '0.17.0'
```

Comment:
I see this helps manage dependencies: https://github.com/ben-manes/gradle-versions-plugin

sounds useful, does it work well in your experience?

Reply:
Yes, it helps list new milestones for all dependencies. We use it in all our company projects.